### PR TITLE
IGAPP-555: Promoting testflight builds to app store connect failing

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: native/fastlane/Fastfile [hunk 1]
```diff
@@ -159,13 +159,13 @@ end
 platform :ios do
   private_lane :apple_auth do |options|
     ensure_env_vars(
-        env_vars: ["APP_STORE_CONNECT_API_KEY_ID", "APP_STORE_CONNECT_API_ISSUER_ID", "APP_STORE_CONNECT_API_KEY"]
+        env_vars: ["APP_STORE_CONNECT_API_KEY_ID", "APP_STORE_CONNECT_API_ISSUER_ID", "APP_STORE_CONNECT_API_KEY_CONTENT"]
```
Comment (klinzo) [inline native/fastlane/Fastfile:162]: With fastlane version v2.175.0 the environment variable APP_STORE_CONNECT_API_KEY is used internally. Setting it manually results in conflicting builds and errors.
https://github.com/fastlane/fastlane/issues/18324

**Comment by klinzo** [comment]
Before merging we have to delete the APP_STORE_CONNECT_API_KEY variable from CircleCI.

**Review by SteffiStuffel — APPROVED** [review]

**Review by maxammann — APPROVED** [review]
Is this PR supposed to solve this issue?

> ITMS-90035: Invalid Signature - The binary with bundle identifier 'de.integreat-app' at path [Integreat.app] contains an invalid signature. Make sure you have signed your application with a distribution certificate, not an ad hoc certificate or a development certificate. Verify that the code signing settings in Xcode are correct at the target level (which override any values at the project level). If you are certain your code signing settings are correct, choose 'Clean All' in Xcode, delete the 'build' directory in the Finder, and rebuild your release target.

**Comment by klinzo** [comment]
> Is this PR supposed to solve this issue?
> 
> > ITMS-90035: Invalid Signature - The binary with bundle identifier 'de.integreat-app' at path [Integreat.app] contains an invalid signature. Make sure you have signed your application with a distribution certificate, not an ad hoc certificate or a development certificate. Verify that the code signing settings in Xcode are correct at the target level (which override any values at the project level). If you are certain your code signing settings are correct, choose 'Clean All' in Xcode, delete the 'build' directory in the Finder, and rebuild your release target.

@maxammann no, it is not. It is just to even get to this point and be able to promote a bulid to app store connect at all. I haven't looked into that issue.

**Pull request merged**